x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -440,16 +447,20 @@ export async function handleLoadMetadataTransformStats(http: HttpStart, store: E
 async function fetchNonExistingPolicies({
   store,
   hosts,
-  http,
+  coreStart,
 }: {
   store: EndpointPageStore;
   hosts: HostResultList['hosts'];
-  http: HttpStart;
+  coreStart: CoreStart;
 }) {
+  if (!canFetchPackageAndAgentPolicies(coreStart.application.capabilities)) {
```

Comment:
I know we continue to try and remove Redux out of our code base, but until then: Would it be possible to pass in or retrieve the endpoint Authz here? Or maybe even remove this code from Redux and create a hook (or use an existing hook) to retrieve the data?